include/miopen/miopen.h

```diff
@@ -5585,6 +5586,29 @@ MIOPEN_EXPORT miopenStatus_t miopenSumForward(miopenHandle_t handle,
 // CLOSEOUT SUM DOXYGEN GROUP
 #endif
 
+#ifdef MIOPEN_BETA_API
+
+/*! @brief Execute a argmax forward layer
+ *
+ * @param handle                   MIOpen handle (input)
+ * @param xDesc                    Tensor descriptor for data input tensor x (input)
+ * @param x                        Data tensor x (input)
+ * @param dim                      Dimensions to reduce argmax. (input)
+ * @param yDesc                    Tensor descriptor for output indice data tensor y (input)
+ * @param y                        Data tensor y (output)
+ * @return                         miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t miopenArgmaxForward(miopenHandle_t handle,
```

Comment:
Can this operation be made part of the TensorOp API, instead of creating a new API call.

Reply:
There is no int32_t dim in miopenOpTensor. Can I modify it and add it?

Reply:
@JehandadKhan I am not sure that it is worth doing. Many miopenOpTensor arguments (`alpha1`, `alpha2`, `beta`, `cDesc` and `C`) will be unused in this case, and we'll need to add a new argument (`dim`).

Reply:
@seungmanhan we can leave it as is